src/mon/mon_types.h

```diff
@@ -37,9 +37,9 @@ enum PAXOS_SVC_TYPE : int {
   PAXOS_NUM
 };
 
-
-inline const char *get_paxos_name(int p) {
-  switch (p) {
+template <const enum PAXOS_SVC_TYPE paxos_svc_type = PAXOS_NUM>
+static constexpr std::string get_paxos_name() {
```

Comment:
i'd suggest just drop this helper. no need to improve a dead function.

Reply:
@tchaikov 
I want to use this function to stringify the enum type member after defining the enum type member.
When extending the paxos service, it's easier to add the enum type member and set the paxos service name at nearby places.

Reply:
i think i'd be better to keep the name of service closer to where the service is defined instead of centralizing them.

Reply:
Got it.
I'll remove get_paxos_name function here.